src/content_script/mastodonInject.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @typedef {Object} VersionNumber
+ * @property {string} major
+ * @property {string} minor
+ * @property {string} patch
+ */
+
+/**
+ * parses a versionNumber string
+ * @returns {VersionNumber}
+ */
+function parseVersionNumber() {
+  const versionElement = document.querySelector(`.link-footer`).textContent
+  const versionNumber = versionElement.match(/v?(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)$/)
+  return versionNumber.groups
+}
+
+
+/**
+ * Replacement onClick handler for Follow button
+ * @param {Event} event 
+ */
+function onClickFollow (event) {
+  event.stopPropagation()
+  event.preventDefault()
+  const username = window.location.pathname.split(`/`).slice(-1)[0]
+  // activate AutoRemoteFollow
+  window.open(`/users/${username}/remote_follow`, `_blank`)
+}
+
+/**
+ * wait for element to appear
+ * @param {string} selector
+ * @param {number} timeout
+ */
+function waitForElement (selector, timeout) {
+  return new Promise((resolve, reject) => {
+    function getElement() {
+      return document.querySelector(selector)
+    }
+
+    const element = getElement()
+    if(element){
+      return resolve(element)
+    }
+
+    const observer = new MutationObserver(() => {
+      const element = getElement()
+      if(element){
```

Comment:
```suggestion
      if (element){
```

Please add spaces after `if`.